docker/build_performous.sh

```diff
@@ -65,15 +65,26 @@ else
 fi
 
 ## Set up some special cmake flags for fedora
-if [ "${ID}" == "fedora" ]; then
+if [[ "${ID}" == "fedora" ]]; then
   EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DUSE_BOOST_REGEX=1"
 fi
 
 ## Set more cmake flags for Debian 10
 # Debian Buster has system Aubio 0.4.5, this is not enough
 # because performous requires a minimum version of 0.4.9.
-if ([ "${ID}" = "debian" ] && [ "${VERSION_ID}" = "10" ]); then
+if ([[ "${ID}" == "debian" && "${VERSION_ID}" == "10" ]]); then
   EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DSELF_BUILT_AUBIO=ALWAYS -DSELF_BUILT_JSON=ALWAYS"
+  echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
```

Comment:
This whole section should be moved to the relevant Dockerfiles for the OS's. The relative Debian commands should be moved to Dockerfile.debian and the 20.04 stuff to Dockerfile.ubuntu